### ISSUE TITLE
question

### ISSUE THREAD

**Issue by shangcai1** [jingyuanli001/RFR-Inpainting#14]
Hello, why does the loss increase when I load the saved parameters after stopping the training and continue training?

**Comment by jingyuanli001** [comment]
I am not sure what's the exact problem in your case. However, if the loss only increases for a few iterations, it's might be due to the frozen batch normalization operators during fine-tuning.
